vault/changelog.early-seed.md

```diff
@@ -2,36 +2,26 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1650339050375
+updated: 1651251246765
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.92.0
+## 0.93.0
 
-Dendron 0.92 has sprouted 🌱
+Dendron 0.93 has sprouted 🌱
 
-**Graph View**: We've improved the loading speed of the graph view. You should notice a drastic improvement in load times-up to 100x in larger workspaces 🚀🚀🚀.
 
-**Tree View**: We've improved the stability of the tree view and fixed an error with `Dendron: Rename Note` that caused the tree view to fall out of sync. 
+### Breaking changes
+This release removes Tree view V2, and the [[enableWebUI|dendron://dendron.dendron-site/dendron.ref.config#enableWebUI]]feature flag will also be removed. We will be focusing on Tree view V1 moving forward. 
```

Comment:
instead of tree view v2, can we change this to the "web view version of tree view"? remove the second sentence "we will be focusing..."